src/screens/APIUtils/APIUtilsTypes.res

```diff
@@ -99,6 +100,7 @@ type userType = [
   | #LIST_ORG
   | #LIST_MERCHANT
   | #LIST_PROFILE
+  | #LIST_ROLES_FOR_INVITE
```

Comment:
url for this #LIST_ROLES_FOR_INVITE usertype is not added in APIUtils ?

Reply:
Duplicate issue in merging two prs